libsrc/supervision/joy/supervision-stdjoy.s

```diff
@@ -0,0 +1,93 @@
+;
+; Standard joystick driver for the Supervision
+;
+
+        .include        "joy-kernel.inc"
+        .include        "joy-error.inc"
+        .include        "supervision.inc"
+
+        .macpack        module
+
+
+; ------------------------------------------------------------------------
+; Header. Includes jump table
+
+        module_header   _supervision_stdjoy_joy
+
+; Driver signature
+
+        .byte   $6A, $6F, $79           ; "joy"
+        .byte   JOY_API_VERSION         ; Driver API version number
+
+; Library reference
+
+        .addr   $0000
+
+; Button state masks (8 values)
```

Comment:
Since the joytick driver interface refactoring a few months ago drivers don't contain state masks anymore. The bit-to-button assignment is now done in the target header file, see e.g. https://github.com/cc65/cc65/blob/master/include/nes.h#L85-L93

If you need/want (for A, B, SELECT, START) button names beyond the the defaults in https://github.com/cc65/cc65/blob/master/include/joystick.h#L59-L66 then you need add the necessary defines as in e.g. https://github.com/cc65/cc65/blob/master/include/nes.h#L95-L103